cmake/GitVersion.cmake

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2014-2019, AEON, The Monero Project
+# 
```

Comment:
This patch is different from the original because the Monero PRs 5632 and 5559 have not yet been picked. Can you merge them in the original order, so that the later merging task becomes easier?

Reply:
Revised to include 5632 5559.